clang/test/dpct/asm/st.cu

```diff
@@ -0,0 +1,21 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none -out-root %T/st %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/st/st.dp.cpp
+// RUN: %if build_lit %{icpx -c -fsycl %T/st/st.dp.cpp -o %T/st/st.dp.o %}
+
+// clang-format off
+#include <cuda_runtime.h>
+
+__global__ void st(int *a) {
+  // CHECK: *a = 111;
+  asm volatile ("st.global.s32 [%0], %1;" :: "l"(a), "r"(111));
```

Comment:
could you list all kind of the test that not support yet as test without checking,  then it easy to understand what is not supported yet. (try to understand what the "partial" mean in inc file.)